airbyte-integrations/connectors/source-ashby/source_ashby/spec.yaml

```diff
@@ -0,0 +1,13 @@
+documentationUrl: https://developers.ashbyhq.com/reference/introduction
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Ashby Spec
+  type: object
+  required:
+    - api_key
+  additionalProperties: true
+  properties:
+    api_key:
+      type: string
+      description: The Ashby API Key.
+      airbyte_secret: true
```

Comment:
What about adding a start date parameter to retrieve data only after a period? I saw endpoints allow this feature: 
![image](https://user-images.githubusercontent.com/5154322/197622805-7ee79408-6429-4d11-a2b2-07367929537e.png)

Reply:
createdAfter is available for application and interview_schedules.

I added the filter for them ✅